### ISSUE TITLE
[BUG] Broken Split Screen If Launched From Recents

### ISSUE THREAD

**Issue by ThatAnnoyingVegan** [Deletescape-Media/Lawnchair#4042]
### Describe the bug

When apps are launched into the split screen from recents (by tapping on the app icon then selecting the split screen mode), 

Both the end corners of the line that separates the two launched apps get flattened and the small flat line in the middle that is used to resize the split screen becomes completely inaccessible and upon swiping up to view the resents the paired apps get separated. 

And sometimes when launching apps like that into spit screen the one I intend to launch doesn't but the app next to it gets launched. 

### Steps to reproduce

1. Go to 'recents menu'
2. Click on 'app icon'
3. Click on 'Split Screen'
4. Launch another app into Split view
5. See error
6. And, Swipe up to recents
7. See error

### Expected behavior

The corners of the split screen shouldn't shape-shift and turn flattened, and apps shouldn't get separated once swiping up to view the recent. 

### Device information

- Device: Poco F1
- OS: Android 13, Proton Plus 13.2
- Security Patch: August 5, 2023
- App version: version 14 Dev 442

**Comment by ThatAnnoyingVegan** [comment]
![Uploading Screenshot_20240201-164238_Next_Player.png…]()

**Comment by ThatAnnoyingVegan** [comment]
I'm unable to attach screen shots my browser crashes... 🥲😵‍💫

**Comment by st0neh** [comment]
Works completely fine here on a Pixel 7a. You appear to be on a build from 2017.

**Comment by PalmDevs** [comment]
> Works completely fine here on a Pixel 7a. You appear to be on a build from 2017.

Nah, it's just that GitHub auto formats `#xxx` to an issue mention. They're on the latest alpha build at the moment which is `CI #442`.

**Comment by Shubhamkr4** [comment]
Same issue here, I can't use split screen at all because in my device split screen gesture don't work on 3rd party launcher so the recents pannel is the only way, bur whenever I tap the split screen and select another app, both the apps blurs out and nothing is usable nor the apps not the bar between them, after swiping up both disappears and the apps seperate as if nothing was happened here

**Comment by ThatAnnoyingVegan** [comment]
> Same issue here, I can't use split screen at all because in my device split screen gesture don't work on 3rd party launcher so the recents pannel is the only way, bur whenever I tap the split screen and select another app, both the apps blurs out and nothing is usable nor the apps not the bar between them, after swiping up both disappears and the apps seperate as if nothing was happened here

Thanks for confirming, have you tested with the taskbar on? Does the described issue persist? In my case yes! I'll test out the latest beta build and will update.

**Comment by Shubhamkr4** [comment]
> > Same issue here, I can't use split screen at all because in my device split screen gesture don't work on 3rd party launcher so the recents pannel is the only way, bur whenever I tap the split screen and select another app, both the apps blurs out and nothing is usable nor the apps not the bar between them, after swiping up both disappears and the apps seperate as if nothing was happened here
> 
> Thanks for confirming, have you tested with the taskbar on? Does the described issue persist? In my case yes! I'll test out the latest beta build and will update.

Turning on the taskbar causes the launcher to crash repeatedly, so its a no go

**Comment by edent** [comment]
Unable to reproduce on 15.Beta 1

Tested split screen with multiple apps, launching from recents, etc. Please re-open if you can still see this bug on the latest nightly.
